.github/workflows/main.yml

```diff
@@ -0,0 +1,273 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  CACHE_SEED: 1
```

Comment:
If this is always `1`, why are we including it in the keys for the cache?

Reply:
Same as [the comment above](https://github.com/pypa/warehouse/pull/8255#discussion_r468752304). TLDR: shouldn't be there. I believe I was using it just as a generic way to update the cache.